pdns/packethandler.cc

```diff
@@ -68,6 +69,7 @@ PacketHandler::PacketHandler():B(s_programname)
   d_sendRootReferral = ::arg().mustDo("send-root-referral")
                             ? ( pdns_iequals(::arg()["send-root-referral"], "lean") ? LEAN_ROOT_REFERRAL : FULL_ROOT_REFERRAL )
                             : NO_ROOT_REFERRAL;
+  d_allowNotifyFrom.toMasks(::arg()["allow-notify-from"] );
```

Comment:
Just tested this, this happens in the PacketHandler constructor, and exceptions are handled higher up: Exiting because communicator thread died with error: Unable to convert 'Habbie' to a netmask